models/_sector/dex/trades/base/platforms/velocimeter_v2_base_base_trades.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+    schema = 'velocimeter_v2_base'
+    , alias = 'base_trades'
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['tx_hash', 'evt_index']
+    , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v2_trades(
+        blockchain = 'base'
+        , project = 'velocimeter'
```

Comment:
If `velocimeter` is a new dex to add, you can update its information in `dex.info`.